tests/helpers.ts

```diff
@@ -0,0 +1,6 @@
+import faker from 'faker';
+import { IEntity } from '../src/entity/models/entity';
+
+export const createFakeEntity = (): IEntity => {
+  return { externalId: faker.random.uuid(), osmId: faker.random.number() };
```

Comment:
`faker.random.number()` define a min value of 0 and max as some big number

Reply:
done